scripts/dlink-sge-signature.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python3
+#
+# append model string and gzip-based signature used by
+# certain D-Link devices manufactured by SGE / T&W.
+#
+# e.g. COVR-P2500 requires this to be appended to the
+# encrypted factory image, while for COVR-C1200 the
+# signature needs to be appended to the unencrypted payload
+#
+# Copyright 2021 OpenWrt.org
```

Comment:
You are not affiliated with OpenWrt, so please don't use this Copyright.

Reply:
Thanks for clarification, this was mostly a boilerplate I found in many other files. So as pointed out by @CodeFetch and considering the discussion in #3449, I conclude that `(C) OpenWrt.org` refers to code contributed by (current or former) members of the project, while non-members simply provide their name and email?

Reply:
From what I know OpenWrt.org is not a legal entity and the preferred way is to use the name of the individual that created the code as the GNU foundation is acting on German copyright law as their legal organization (gpl-violations.org) which is a German organization (as German copyright law is one of the strictest and easily enforceable). I think there was a discussion on the mailing list with the result that new code should always contain the individual's name, but that the existing copyright remarks (with OpenWrt.org) should be left untouched (not adding the names of the creators afterwards).

@adschm I think that someone stated in that discussion that OpenWrt should also be mentioned to make it easier to search for copyrighted material in OEM GPL tarballs... So maybe just add a comment like "This code is part of OpenWrt" or something?! It would be nice if there was a hint on that in the contribution guidelines...